fluid/PaddleCV/image_classification/infer.py

```diff
@@ -7,11 +7,11 @@
 import sys
 import paddle
 import paddle.fluid as fluid
-import models
+import models_name as models
 import reader
 import argparse
 import functools
-from models.learning_rate import cosine_decay
+from utils.learning_rate import cosine_decay
```

Comment:
remove this line. Infrence does not need lr.

Reply:
OK